src/components/inputs/select/dropdown.component.tsx

```diff
@@ -69,6 +69,7 @@ const Dropdown: React.FC<FormFieldInputProps> = ({ field, value, errors, warning
             invalidText={errors[0]?.message}
             items={items}
             itemToString={itemToString}
+            label=""
```

Comment:
Shouldn't this have a value?

Reply:
The [Dropdown](https://react.carbondesignsystem.com/?path=/docs/components-dropdown--overview) component accepts both `label` and  `titleText` props:

![CleanShot 2025-04-11 at 2  05 02@2x](https://github.com/user-attachments/assets/8b134348-7f27-4e6d-af89-bb75d78eeab8)

The `titleText` is what gets used as a label for the `Dropdown`. The `label` is defined as:

> a generic `label` that will be used as the textual representation for what this field is for

It defaults to an empty string. It seems fine to me to leave it as an empty string because whatever we set it to is what gets rendered as the default placeholder in the input. Additionally, because it's a required prop, omitting it raises a missing prop warning from the TypeScript compiler.